README.rst

```diff
@@ -2,56 +2,107 @@
 Nurse Scheduling
 ================
 
-THIS IS A WORK IN PROGRESS! ONLY TEMPLATE FILES HERE AT THE MOMENT.
+This is a demo of a nurse scheduling algorithm developed by Ikeda, Nakamura
+and Humble. The nurse scheduling problem seeks to find an optimal assignment
+for a group of nurses, under constraints of scheduling and personnel.
+The example is a simplified representation of a real-world nursing facility,
+but the model includes many parameters which are not explored here.
 
-Short blurb on what this demo does.
+There are three overall constraints in the full problem:
 
-If your demo produces a visualization, consider displaying an image of it here.
+1) Both upper and lower limits on the number of breaks.
+2) The number of nurses on duty for each shift slot.
+3) For each individual nurse, upper and lower limits on the time interval 
+between days of duty.
 
-.. image:: dwave_logo.png
+The overall requirements combine to ensure that there are enough nurses
+on duty at all times, and that the individual nurses are not expected to work
+more than they can.
 
+The "hard shift" constraint requires that at least one nurse is assigned for
+each working day.
+
+The "hard nurse" constraint ensures that no nurse works two or more 
+consecutive days.
+
+The "soft nurse" constraint requires that all nurses should have roughly
+even work schedules.
+
+Running the demo results in the following output:
+
+    Size  33
+
+    Energy  0.5999999999999694
+
```

Comment:
@joelpasvolsky, it's just a print statement. The energy could be different, but it just turns out that way in this case. I'm trying not to "finalize" the problem, because it needs more research.

Reply:
Seeing as it's not clear in the README that this represents a literal output of a print statement, might as well be human readable, no?

Reply:
OK, @joelpasvolsky, I need to be clear, because I'm not understand your thinking. To me, this is a print statement, and it's OK that it is a floating-point number. Are you thinking that it should be a formatted statement, so that it is more readable?

Reply:
No, now it is clear. But I think under "The results show the following" it would be good to add a bullet on the meaning of the `0.6`  for the soft constraint (basically for those who aren't conversant in hard vs soft constraints). It's enough to explain how this number represents a degree of compliance with the soft constraint as opposed to the yes/no violation of hard constraints